userspace/falco/application.h

```diff
@@ -193,6 +193,11 @@ class application {
 		return !m_options.trace_filename.empty();
 	}
 
+	inline bool is_gvisor_enabled() const
+	{
+		return m_state->inspector->is_gvisor();
```

Comment:
Note for the reviewer: the astute reader will have noticed that the boolean method `is_gvisor()` does not exist yet in falco libs, so I'm going to add it. I think it's the clean way to implement this check but if maintainers and contributors have better ideas I can change the implementation.

Reply:
Now it is implemented.